packages/web/src/js/FileUploader.ts

```diff
@@ -278,7 +287,7 @@ class FileUploader extends BaseComponent {
   removeFromQueue(name: string) {
     if (this.fileQueue.has(name)) {
       EventHandler.trigger(this.wrapper, EVENT_UNQUEUE_FILE, { fileQueue: this.fileQueue });
-      const itemElement = SelectorEngine.findOne(`li#${name}`);
+      const itemElement = SelectorEngine.findOne(`#\\${name}`);
```

Comment:
🤔

Reply:
I have introduced a unique prefix for the file name and this prefix can have a number in it. So this escapes it.

Reply:
Made this better and move the unique string after the `file_` prefix.